builder_test.go

```diff
@@ -1003,3 +1003,66 @@ func TestSendWithReversePath(t *testing.T) {
 		t.Errorf("msg bytes did not contain text body %q", text)
 	}
 }
+
+func TestEmptyTo(t *testing.T) {
+	sender := &mockSender{}
+	from := "from@example.com"
+	text := []byte("test text body")
+	ret := "return@example.com"
+	a := enmime.Builder().
+		Text(text).
+		From("name", from).
+		Subject("foo").
+		To("", "")
+
+	err := a.SendWithReversePath(sender, ret)
+	if err == nil {
+		t.Fatal("Expected error, got nil")
+	}
+
+	if err.Error() != "no recipients (to, cc, bcc) set" {
+		t.Fatalf("Unexpected error, wanted 'no recipients (to, cc, bcc) set' got %s", err)
```

Comment:
We could add the text of the error to https://github.com/jhillyerd/enmime/blob/master/error.go and use it in both places.